Demandware/Experian_EDQ_SGJC/cartridge/static/default/js/EDQUtils.js

```diff
@@ -258,4 +268,211 @@ function countryAlpha3(vAlphaTwo) {
             vAlphaThree = countryDict[i].value;
     }
     return (vAlphaThree) ? vAlphaThree : vDefaultCountry;
+}
+
+for (var i = 0; i < inputSelector.length; i++)
+{
+    if (inputSelector[i].hasAttribute('id')) {

```

Comment:
In order to avoid multiple-level indentation, you could fail fast - eg if (!condition) { break; } then continue with the logic that needs to be performed inside the for statement. The same applies for all the subsequent similar cases.